sippy-ng/src/components/Sidebar.js

```diff
@@ -62,18 +62,20 @@ export default function Sidebar(props) {
     }
   }, [props])
 
-  const handleBugzillaOpen = () => {
-    setBugzillaOpen(true)
-  }
-
-  const handleBugzillaClose = () => {
-    setBugzillaOpen(false)
-  }
-
   function handleClick(id) {
     setOpen((prevState) => ({ ...prevState, [id]: !prevState[id] }))
   }
 
+  function reportAnIssueURI() {
+    const description = `Describe your feature request or bug:\n\n
+    
+    Relevant Sippy URL:\n
+    ${window.location.href}\n\n`
+    return `https://issues.redhat.com/secure/CreateIssueDetails!init.jspa?pid=12323832&issuetype=17&description=${safeEncodeURIComponent(
+      description
```

Comment:
default priority is shown as `Blocker`.  I tried these and could not find `Normal`.  Maybe `Minor` is the way to go with `&priority=4`?

We can also say default priority is set to whatever we pick so they can choose it consciously.

```
  4 = minor
  3 = major
  2 = critical
  1 = blocker
  0 = invalid
```

Reply:
![image](https://github.com/openshift/sippy/assets/429763/e627bfb3-2168-41a6-84fb-8a83fd41b644)

It's 10200 lol